packages/lib/services/interop/Module.ts

```diff
@@ -10,6 +10,7 @@ interface BaseMetadata {
 	fileExtensions: string[];
 	description: string;
 	isDefault: boolean;
+	separator: boolean;
```

Comment:
Because `separator` relates to a menu GUI in which the module can be displayed (while the other properties relate to the importer itself), it may make sense to either
1. leave a comment explaining what `separator` means here (i.e. that `separator` isn't a property of the importer, but of how it should be displayed in the GUI)
2. rename `separator` to something like `lastOfGroup`